components/wishlist/WishlistButton/WishlistButton.tsx

```diff
@@ -34,7 +34,7 @@ const WishlistButton: FC<Props> = ({
       item.product_id === productId &&
       item.variant_id === variant.id
   )
-
+  
```

Comment:
```suggestion
 
```

Reply:
Are you suggesting that we add a space on line 37 here?